src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -12,6 +12,8 @@
 import org.joda.time.DateTimeZone;
 
 import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.logic.api.Logic;
```

Comment:
Basically i need a instructor list to check their displaytostudent status, Can i used Logic dependency into InstructorAttributes then?